docs/getting_started.rst

```diff
@@ -136,7 +136,7 @@ a private network.
     from cloudbridge.cloud.interfaces.resources import TrafficDirection
     fw = provider.security.vm_firewalls.create(
         label='cloudbridge-intro', description='A VM firewall used by
-        CloudBridge', network=net.id)
+        CloudBridge', network_id=net.id)
```

Comment:
Shouldn't this be network?

Reply:
Yes! This was probably before we patched that, sorry!